crates/biome_module_graph/src/js_module_info/collector.rs

```diff
@@ -114,6 +114,10 @@ pub(super) enum JsCollectedExport {
         /// Reference to the type being exported.
         ty: TypeReference,
     },
+    ExportNamedDefault {
```

Comment:
This variant is used for `export default myIdent` syntax, so it correctly resolves to the binding for `myIdent`. This ensures that the JsDoc for `myIdent` is also found. The same issue would have plagued `noPrivateImports`.